docs/help-center/using-famedly/chat-backup-verification.mdx

```diff
@@ -0,0 +1,120 @@
+    <li>One of the devices will now display a QR-Code, the other device will open a QR-Code scanner.</li>
+    <li>Please scan the displayed QR-Code.</li>
+    <li>If by some reason you are not able to scan the QR-Code, you can tap on <b>Compare numbers instead of QR-Code</b>.</li>
+    <li>After scanning the QR-Code, you will be asked if it was really you who scanned the QR-Code on your already active device. Tap <b>Yes, it was me</b> to complete the verification.</li>
+  </ol>
+  </TabItem>
+</Tabs>
+
+## Number verification
+
+If both devices are desktop devices or web browsers you get asked for number verification instead of QR-Verification automatically. Just compare the numbers and click <b>They match</b> on both devices.  
+
+### Use number verification with mobile device
+
+Sometimes you might need to use number verification instead of QR-Verification (e.g. if your phones camera is broken), therefore you can also follow these steps. 
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Accept the verification request by clicking on <b>Continue</b> in the <b>New verification request</b> dialog.</li>
+    <li>If the other device is a mobile device, that device will now open a QR-Code scanner. Tap <b>Compare numbers instead of QR-Code</b>.</li>
+    <li>Compare the numbers on the devices and press **They match** on both devices.</li>
+  </ol>
+  </TabItem>
+  <TabItem value="mobile" label="Mobile">
+  <ol>
+    <li>Accept the verification request by tapping on <b>Continue</b> in the <b>New verification request</b> dialog.</li>
+    <li>One of the devices will now display a QR-Code, the other device will open a QR-Code scanner.</li>
+    <li>Tap **Compare numbers instead of QR-Code** below the QR-Code scanner.</li>
+    <li>Compare the numbers on the devices and press **They match** on both devices.</li>
+  </ol>
+  </TabItem>
+</Tabs>
+
+
+## Verification without other devices
+
+If you have no other device logged in or you lost access to the other devices, you can still restore your backup by entering your passphrase. This passphrase was shown to you when you activated your chat backup and it begins with the letters <b>Es</b>.  
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ol>
+    <li>Click <b>No access to other devices?</b>.</li>
+    <li>Enter your passphrase into the input field.</li>
+    <li>Click <b>Restore backup</b>.</li>
+  </ol>
+  </TabItem>
+  <TabItem value="mobile" label="Mobile">
+  <ol>
+    <li>Tap <b>No access to other devices?</b>.</li>
+    <li>Enter your passphrase into the input field.</li>
+    <li>Tap <b>Restore backup</b>.</li>
+  </ol>
+  </TabItem>
+</Tabs>
+
+## Forgotten passphrase?
+
+If you forgot your passphrase and have no other devices available, you can reset your passphrase. 
+
+:::danger
+You will lose access to older messages if you reset your passphrase. Please only reset your passphrase if you're sure that you have no access to it. 
```

Comment:
'to previous messages' or different wording 'Your account will be reset and your message content will no longer be available.'